src/noise/quantum_error.hpp

```diff
@@ -387,6 +390,20 @@ void QuantumError::load_from_json(const json_t &js) {
   set_circuits(circuits, probs);
 }
 
+
+bool QuantumError::pauli_only(void) const 
```

Comment:
According to the local discussion with @itoko -san, it is bbetter to rename `pauli_only()` with `is_mixed_pauli()`.